tck/pom.xml

```diff
@@ -17,14 +17,16 @@
 
     <dependencies>
         <dependency>
-            <groupId>io.github.a2asdk</groupId>
+            <groupId>${project.groupId}</groupId>
             <artifactId>a2a-java-sdk-reference-jsonrpc</artifactId>
-            <version>${project.version}</version>
         </dependency>
         <dependency>
             <groupId>io.github.a2asdk</groupId>
             <artifactId>a2a-java-sdk-reference-grpc</artifactId>
-            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.github.a2asdk</groupId>
+            <artifactId>a2a-java-sdk-reference-rest</artifactId>
```

Comment:
The additional interfaces in the `AgentCardProducer` in the `tck` directory also needs to be updated to declare support for REST.

Reply:
We'll also want to update the run-tck.yml file to:

* Add the SUT_REST_URL here https://github.com/a2aproject/a2a-java/blob/main/.github/workflows/run-tck.yml#L58 (9999 is used for JSONRPC, 9000 for gRPC)
* Add rest to the tested transports here https://github.com/a2aproject/a2a-java/blob/main/.github/workflows/run-tck.yml#L96 (we haven't tried running the TCK with Rest before so there might be some issues)

Reply:
@ehsavoie Just to check, was this addressed?